pkg/preflight/checks.go

```diff
@@ -200,6 +205,18 @@ func (c MemoryCheck) Run() (string, error) {
 	return "", nil
 }
 
+func (c DiskCheck) Run() (msg string, err error) {
+	disks, err := utilGetUniqueDisks()
+	if err != nil {
```

Comment:
Generally `util.GetUniqueDisks` did not get any error. Should we also generate the fatal error msg here?